### PR TITLE
Fix tests, relative paths, slashed paths, other stuff

### PR DESCRIPTION
**Pull request by flickyiyo**

### CODE REVIEW

File: src/fence_collection.rs [hunk 1]
```diff
@@ -30,3 +33,57 @@ impl FenceCollection {
...
+    #[test]
+    fn test_get_fences_for_path() {
+        let fence_collection = FenceCollection {
+            fences_map: map!(
+                "some/a/b/fence.json" => parse_fence_str(
+                    r#"{"tags": ["b"]}"#,
+                    &RelativePathBuf::from("path/to/protected/fence.json")
+                ).unwrap(),
+                "some/fence.json" =>  parse_fence_str(r#"{"tags": ["root"]}"#, &RelativePathBuf::from("path/to/protected/fence.json")).unwrap(),
+                "some/fucking/fence.json" =>  parse_fence_str(r#"{"tags": ["fucking"]}"#, &RelativePathBuf::from("path/to/protected/fence.json")).unwrap()
```
Comment (Adjective-Object) [inline src/fence_collection.rs:69]:
```suggestion
                "some/other/fence.json" =>  parse_fence_str(r#"{"tags": ["other"]}"#, &RelativePathBuf::from("path/to/protected/fence.json")).unwrap()
```

File: src/fence_collection.rs [hunk 2]
```diff
@@ -30,3 +33,57 @@ impl FenceCollection {
...
+
+        assert_eq!(
+            fence_collection.get_fences_for_path(&Path::new("some/fucking/file.ts")),
+            vec![
+                fence_collection.fences_map.get("some/fucking/fence.json").unwrap(),
+                fence_collection.fences_map.get("some/fence.json").unwrap(),
+            ],
+            "should return multiple fences for file with multiple fences",
+        );
+    }
```
Comment (Adjective-Object) [inline src/fence_collection.rs:88]:
```suggestion
        assert_eq!(
            fence_collection.get_fences_for_path(&Path::new("some/other/file.ts")),
            vec![
                fence_collection.fences_map.get("some/other/fence.json").unwrap(),
                fence_collection.fences_map.get("some/fence.json").unwrap(),
            ],
            "should return multiple fences for file with multiple fences",
        );
    }
```

File: src/get_imports/import_path_visitor.rs [hunk 3]
```diff
@@ -25,6 +25,35 @@ impl ImportPathVisitor {
...
+            
+            if let Some(imports) = self.imports_map.get_mut(&source) {
+                specifiers.drain().for_each(|x| {
+                    imports.insert(x);
+                });
+            } else {
+                self.imports_map.insert(source, HashSet::from_iter(specifiers));
+            }
+        }
+        // export.
```
Comment (Adjective-Object) [inline src/get_imports/import_path_visitor.rs:54]:
```suggestion
```

File: src/get_imports/import_path_visitor.rs [hunk 4]
```diff
@@ -136,6 +165,27 @@ mod test {
...
+        let fm = cm.new_source_file(
+            FileName::Custom("test.ts".into()),
+            r#"export { default as a, foo as bar } from './foo'"#.to_string(),
+        );
+
+        let mut parser = create_test_parser(&fm);
+
+        let mut visitor = ImportPathVisitor::new();
+        let module = parser.parse_typescript_module().unwrap();
+        // println!("{:?}", module.body);
```
Comment (Adjective-Object) [inline src/get_imports/import_path_visitor.rs:180]:
```suggestion
```

**Review by Adjective-Object — CHANGES_REQUESTED** [review]
nits and cleanliness

File: src/fence_collection.rs (hunk 2 above)
Comment (flickyiyo) [reply src/fence_collection.rs:88]: boo!

mood-boo

**Pull request merged**